### ISSUE TITLE
A1112 exceptions with some proprietary fonts

### ISSUE THREAD

**Issue by HinTak** [Microsoft/Font-Validator#41]
It takes nearly 8 complete days to run the glyf test on my private stash of old cjk fonts. (and lost the run three times, so lost about 10 days of works I think, in the middle... so it took nearly 3 weeks). The fonts which shows A1112 exceptions are:

```
avbyv.ttf
avfv.ttf
avmbv.ttf
avmyv.ttf
avsbv.ttf
avssbv.ttf
avsv.ttf
bhei00b.ttf
bhei00l.ttf
bhei01b.ttf
bhei01l.ttf
bshei01b.ttf
bsmi00lp.ttf
dcinl5.ttc
dffn_3.ttc
dffn_j5.ttc
dffrs3.ttc
dfkaie.ttc
dfttlx3.ttc
dwfszk.ttf
dwktzk.ttf
dwstzk.ttf
gbsn00l.ttf
ggtr00l.ttf
glei00m.ttf
gsin00m.ttf
gsun00l.ttf
gtr00bp.ttf
gtr00lp.ttf
gtr00mp.ttf
htst3.ttf
jdfkanting.ttf
jdfxingshu.ttf
jdfzhongb.ttf
jdjhte.ttf
jdjlibian.ttf
jdjste.ttf
jdjxinshu.ttf
jdjychao.ttf
jdtuan.ttf
kochi-mincho-subst.ttf
min08mp.ttf
sgtr00b.ttf
simsunb.ttf
ska00bp.ttf
ssin00m.ttf
ssun00l.ttf
sun00mp.ttf
swei00b.ttf
tgtr00l.ttf
toor00u.ttf
tovr00u.ttf
tsin00m.ttf
wdr.ttf
wei.ttf
wf2.ttf
wf3.ttf
wf4.ttf
wh1.ttf
wh4.ttf
wh5.ttf
wh6.ttf
wk2.ttf
wk3.ttf
wktv.ttf
wl1.ttf
wodr.ttf
wov.ttf
wpop1.ttf
wr1.ttf
wr2.ttf
wr3.ttf
wr5.ttf
ws1.ttf
ws2.ttf
ws3.ttf
wsf.ttf
yka00hp.ttf
简启体.ttf
繁启体.ttf
```

and covers almost all of the glyf test error codes:

```
    347 A1112
    262 E1100
     72 E1102
2116848 E1110
 596493 E1111
2405587 E1112
    112 E1113
      1 E1114
      4 E1115
      3 E1117
   1497 E1119
    509 E1120
     60 E1123
 700263 I1111
    438 W1100
 445094 W1110
2231479 W1111
3336672 W1112
    280 W1113
```

what is interesting is that there is one more sub-type:

```
ValidateCompComponentInters Exception: Exception GMath: class: Bez2D method: ParamFromSupport details: NOT IMPLEMENTED  at NS_Glyph.Glyph.CallGV (TypeGV typeGV) 
ValidateSimpContInters Exception: Exception GMath: class: Bez2D method: ParamFromSupport details: NOT IMPLEMENTED  at NS_Glyph.Glyph.CallGV (TypeGV typeGV) 
ValidateSimpContInters Exception: Exception GMath: class: Equation method: RootsAll details: Degree 3  at NS_Glyph.Glyph.CallGV (TypeGV typeGV) 
ValidateSimpContMisor Exception: Exception Glyph: class: Glyph method: ValidateSimpContMisor  at NS_Glyph.Glyph.CallGV (TypeGV typeGV) 
```

**Comment by HinTak** [comment]
One more sub-types than
https://github.com/Microsoft/Font-Validator/issues/40

**Comment by HinTak** [comment]
Fixed in upcoming v2.1
